draft-ietf-taps-arch.md

```diff
@@ -89,15 +97,21 @@ This document describes an architecture for exposing transport protocol features
 
 The goal of the Transport Services architecture is to provide a common, flexible, and reusable interface for transport protocols. As applications adopt this interface, they will benefit from a wide set of transport features that can evolve over time, and ensure that the system providing the interface can optimize its behavior based on the application requirements and network conditions, without requiring changes to the applications. This flexibility enables faster deployment of new features and protocols. It can also support applications by offering racing and fallback mechanisms, which otherwise need to be implemented in each application separately.
 
 This document was developed in parallel with the specification of the Transport Services API {{!I-D.ietf-taps-interface}} and Implementation Guidelines {{?I-D.ietf-taps-impl}}. Although following the Transport Services Architecture does not require that all APIs and implementations are identical, a common minimal set of features represented in a consistent fashion will enable applications to be easily ported from one system to another.
 
 ## Background
 
-The Transport Services architecture is based on the survey of services provided by IETF transport protocols and congestion control mechanisms {{?RFC8095}}, and the distilled minimal set of the features offered by transport protocols {{?I-D.ietf-taps-minset}}. These documents identified common features and patterns across all transport protocols developed thus far in the IETF.
+The BSD Unix Socket prgramming interface and its SOCK_STREAM abstraction was a revolution in simplicity: by bringing network sockets into the UNIX programming model, it enabled anyone who knew how to write programs that dealt with sequential-access files to also write network applications. The simplicity of this API is a key reason the	Internet won the protocol wars {{PROTOCOL-WARS}} of the 1980s. SOCK_STREAM is	tied to the Transmission Control Protocol (TCP), specified in 1981 {{?RFC0793}}.	
+TCP has scaled remarkably well over the past three and a half decades, but its total ubiquity has hidden an uncomfortable fact: the network is not really a	file, and stream abstractions are too simplistic for many modern application	
+programming models.	
+
+In the meantime, the nature of Internet access, and the variety of Internet	transport protocols, is evolving. The challenges that new protocols and access paradigms present to the Socket API and to programming models based on them	
+inspire the design principles of a new approach.
+Therefore, the new Transport Services architecture is based on the survey of services provided by IETF transport protocols and congestion control mechanisms {{?RFC8095}}, and the distilled minimal set of the features offered by transport protocols {{?I-D.ietf-taps-minset}}. These documents identified common features and patterns across all transport protocols developed thus far in the IETF.
 
```

Comment:
I am fine leaving this extension out and keeping it simple. If we do decide to keep the text. Then I would like "new Tarnsport Services architecture" to be just "Transport Services architecture. I do not like the "new" for reasons previously commented on.